lms/djangoapps/support/static/support/jsx/single_support_form.jsx

```diff
@@ -183,8 +193,67 @@ class RenderForm extends React.Component {
     );
   }
 
+  async handleInputChange(event) {
+    event.preventDefault();
+    const queryString = event.target.value;
+    if (queryString.length > 2) {
+      let suggestions = [];
+      await fetch(`${this.learnerSupportCenterURL}/hc/api/internal/instant_search.json?query=${queryString}&locale=en-us`, { headers: {
+        'Content-Type': 'application/json',
+      }}).then((response) => {
+        suggestions = response.json.results;
+        this.setState({ suggestions });
+      });
+    } else {
+      this.setState({
+        suggestions: [],
+        activeSuggestion: 0,
+      });
+    }
+  }
+
+  onKeyDown(event) {
+    const { activeSuggestion, suggestions } = this.state;
+    const enterKeyCode = 13,
+      upArrowKeyCode = 38,
+      downArrowKeyCode = 40;
+
+    if (event.keyCode === enterKeyCode) {
+      window.location.href = this.learnerSupportCenterURL + suggestions[activeSuggestion].url;
+    } else if (event.keyCode === upArrowKeyCode) {
+      (activeSuggestion === 0) ?
+      this.setState({ activeSuggestion: suggestions.length - 1 }) :
+      this.setState({ activeSuggestion: activeSuggestion - 1 });
+    } else if (event.keyCode === downArrowKeyCode) {
+      (activeSuggestion + 1 === suggestions.length) ?
+      this.setState({ activeSuggestion: 0 }) :
+      this.setState({ activeSuggestion: activeSuggestion + 1 });
+    }
+  }
+
+  handleBlur(event) {
+    if (!this._ignoreBlur) {
+      this.setState({
+        suggestions: [],
+        activeSuggestion: 0,
+      });
+    }
+  }
+
+  handleSearchButton(query) {
+    const queryString = query.replace(' ', '+');
+
+    window.location.href = `${this.learnerSupportCenterURL}/hc/en-us/search?utf8=✓&query=${queryString}`;
```

Comment:
Typo in url(after utf-8).

Reply:
that ✓ is necessary for the search results.

Reply:
use encoded value instead of the actual char(do verify the api call works with encoded value)